DEVELOPMENT.md

```diff
@@ -0,0 +1,178 @@
+
+`npm` comes as part of the Node project, which you can download [here][Node.js]. The LTS (long-term support) version should be fine.
+
+On Windows, you'll be asked if you want to install tools for native modules. We don't have any native modules in this project, but you'll probably want to check that box anyway, in case you want to install any in the future. (Installing tools for native modules may take a while.)
+
+![The npm installer asking whether you'd like tools for native modules, with the checkbox checked][i:Windows, installing npm]
+
+## Telling `npm` how to talk to GitHub Packages
+
+GitHub Packages is where we keep all of the images for the website, along with some data about flowers and bees. 
+
+For `npm` read from GitHub Packages, you'll need to provide it with a personal access token.
+
+### Generating the personal access token
+
+You can create a personal access token by going to <https://github.com/settings/tokens> and clicking "Generate new token". Under "Scopes", you'll want to check `repo`, `read:packages`, `write:packages`, and `delete:packages`:
+
+![The "New personal access token" page on GitHub, with those four scopes checked.][i:New personal access token]
+
+Then, you can click "Generate token" at the bottom of the page. You'll get a token that looks like `745b0dcc742dedf87c67ff075f65eb87295e205f`. Keep that around.
+
+### Giving that personal access token to `npm`
+
+Next, open a command prompt (Terminal.app on the Mac; PowerShell on Windows) and run
+
+```sh
+npm login --registry=https://npm.pkg.github.com
+```
+
+ * When it asks for a username, enter your GitHub username.
+ * When it asks for a password, paste in the personal access token (***not*** your GitHub password).
+ * When it asks for your email, enter your GitHub email. 
+
+![Powershell after running "npm login --registry=https://npm.pkg.github.com". The output "Logged in as helloworld12321demo on https://npm.pkg.github.com/" is shown.][i:Logging in to npm]
+
+Some sticking-points:
+
+1. For security reasons, what you type into the "password" field is invisible.
+2. To paste text into PowerShell, right-click once.
+3. On github.com/settings/tokens, there personal access token has a space after it&mdash;be careful that you don't copy that! :upside_down_face:
+
+![github.com/settings/tokens with the extra space character highlighted][i:The gosh-darn whitespace]
+
+If you run into trouble, you can always fall back to typing the personal access code by hand.
+
+## Installing all of the third-party libraries
+
+Buzz About uses code from several third-party packages. You can install all of these packages at once using `npm`.
+
+Open up the command prompt. First, you'll want to navigate to the folder where you downloaded Buzz About. You can do this by running
+
+```PowerShell
+Set-Location C:\wherever\you\downloaded\buzz-about
```

Comment:
Having a separate command for Windows is OK but I would still use `cd` as that is simply more standardized and works in both Command Prompt and PowerShell. Also people's terminals (powershell or bash) usually start in the home folder by default so the user can likely use a relative path.

Reply:
Good point about the compatibility with the Command Prompt—I'll change that :)

Re: relative paths, you're totally right that that's more idiomatic–—I'm just using an absolute path because that's probably more familiar to the average user, who might have seen it in file preferences or something. (And if people do know about relative paths, they probably don't need me to explain `cd` to them :upside_down_face:)